parm/config/gfs/config.aero

```diff
@@ -34,10 +34,17 @@ esac
 # Biomass burning emission dataset. Choose from: GBBEPx, QFED, NONE (default)
 AERO_EMIS_FIRE=QFED
 
+export AERO_DIAG_TABLE="${AERO_DIAG_TABLE:-${HOMEgfs}/parm/ufs/fv3/diag_table.aero}"
+export AERO_FIELD_TABLE="${AERO_FIELD_TABLE:-${HOMEgfs}/parm/ufs/fv3/field_table.aero}"
+AERO_EMIS_FIRE="${AERO_EMIS_FIRE:-none}"
+export AERO_EMIS_FIRE=${AERO_EMIS_FIRE,,}
+export AERO_CONFIG_DIR="${AERO_CONFIG_DIR:-${HOMEgfs}/parm/ufs/chem}"
+export AERO_INPUTS_DIR="${AERO_INPUTS_DIR:-}"
```

Comment:
This is `config.aero` and there is no higher level where these values are being set or should be set; thus they should not have pass-through options
```suggestion
export AERO_DIAG_TABLE="${HOMEgfs}/parm/ufs/fv3/diag_table.aero"
export AERO_FIELD_TABLE="${HOMEgfs}/parm/ufs/fv3/field_table.aero"
export AERO_EMIS_FIRE="none"
export AERO_CONFIG_DIR="${HOMEgfs}/parm/ufs/chem"
export AERO_INPUTS_DIR=""
```

Can the `chem` in `AERO_CONFIG_DIR` be named `aerosols` or `aero`?

Reply:
Changed it to `gocart`, as that is consistent with other directories `mom6` and `fv3`.